pim/data/ir_module_category_data.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="module_category_pim" model="ir.module.category">
+        <field name="name">PIM</field>
```

Comment:
I´d move this category and the base menu entry to a base_pim module so that it can be shared w/ all pim modules